CHANGELOG.md

```diff
@@ -2,6 +2,19 @@
 
 # UNRELEASED
 
+## DFX
+
+### feat: use OS-native keyring for pem file storage
+
+If keyring integration is available, PEM files are now by default stored in the OS-provided keyring.
+If it is not available, it will fall back on the already existing password-encrypted PEM files.
```

Comment:
It looks like the `default` identity is still stored unencrypted at `~/.config/dfx/identity/default/identity.pem`.  Is that intentional, and if so, should we mention that here?  It doesn't match the behavior described in this changelog entry.

Reply:
Mentioning it explicitly now. Yes, this is intentional because we don't want to touch keyring in any default flows. Otherwise this will hang 99.9% of all CI uses of dfx because it will be stuck on waiting for approval.